library/libwaku.nim

```diff
@@ -54,7 +54,7 @@ proc handleRequest(
     callback: WakuCallBack,
     userData: pointer,
 ): cint =
-  waku_thread.sendRequestToWakuThread(ctx, requestType, content, callback, userData).isOkOr:
+  waku_context.sendRequestToWakuThread(ctx, requestType, content, callback, userData).isOkOr:
```

Comment:
can't it just be `context`? we are in the waku lib, a `waku context` seems redundant.